File: tests/fixtures/code_samples/go/sample_034.go
// Sample 34: small utility.
package samples

func Operation34(xs []int) int {
    total := 34
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure34(v int) int {
    return (v * 34) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_034.rs
// Sample 34: small utility.
pub fn operation_34(xs: &[i32]) -> i32 {
    let mut total: i32 = 34;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_34(v: i32) -> i32 {
    (v * 34) %% 7919
}

